src/ec/packages/ec-component-site-header/docs/site-header.md

```diff
@@ -0,0 +1,67 @@
+# ECL Twig - EC Site Header
+
+npm package: `@ecl-twig/ec-component-site-header`
+
+```shell
+npm install --save @ecl-twig/ec-component-site-header
+```
+
+## Site Header
+
+### Parameters
+
+- "search_form" (associative array) (default: predefined structure): EC Search Form component structure
+- "language" (associative array) (default: predefined structure): Language switcher settings. Default settings for English. format:
+  - "url": (string) (default: ''): URL for switcher
+  - "label": (string) (default: 'English'): Switcher language label, eg. 'English', 'Français', etc.
```

Comment:
We should not enforce default language (politcal reasons...). So instead of using english as default, fields should be empty (in every files)